src/test/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacadeIT.java

```diff
@@ -74,6 +75,8 @@ public void getEventsTest() {
     // NOTE: We may end up having more events in the dataset than the limit specified in the call.
     //       In this case, we need to check for the limit up here and then check if the response object tells us
     //       that there are more, and how many there are.
+    assertEquals(8, results.stream().filter(event -> event.getPrivateEvent() == null || !event.getPrivateEvent()).count());
```

Comment:
It would be more readable for both these lines to move the `results.stream().filter...` out into a variable with a name that explains its intention. That way it's more readable.